site/components/grid/style.css

```diff
@@ -129,7 +126,7 @@
  */
 
 .size1of12 {
-  flex-basis: auto; /* 1 */
+  /* flex-basis: auto; 1 */
```

Comment:
was this intentional? there does not seem to be much point to having this as a comment, if the rule is not needed we should just remove it.